ddoc.dd

```diff
@@ -731,70 +736,50 @@ DDOC = $(LT)html$(GT)$(LT)head$(GT)
+DDOC_BUGS             = $(DOLLAR)(RED BUGS:)$(DOLLAR)(BR)$(DOLLAR)0$(DOLLAR)(BR)$(DOLLAR)(BR)
+DDOC_COPYRIGHT        = $(DOLLAR)(B Copyright:)$(DOLLAR)(BR)$(DOLLAR)0$(DOLLAR)(BR)$(DOLLAR)(BR)
+DDOC_DATE             = $(DOLLAR)(B Date:)$(DOLLAR)(BR)$(DOLLAR)0$(DOLLAR)(BR)$(DOLLAR)(BR)
+DDOC_DEPRECATED       = $(DOLLAR)(RED Deprecated:)$(DOLLAR)(BR)$(DOLLAR)0$(DOLLAR)(BR)$(DOLLAR)(BR)
+DDOC_EXAMPLES         = $(DOLLAR)(B Examples:)$(DOLLAR)(BR)$(DOLLAR)0$(DOLLAR)(BR)$(DOLLAR)(BR)
+DDOC_HISTORY          = $(DOLLAR)(B History:)$(DOLLAR)(BR)$(DOLLAR)0$(DOLLAR)(BR)$(DOLLAR)(BR)
+DDOC_LICENSE          = $(DOLLAR)(B License:)$(DOLLAR)(BR)$(DOLLAR)0$(DOLLAR)(BR)$(DOLLAR)(BR)
+DDOC_PARAMS           = $(DOLLAR)(B Params:)$(DOLLAR)(BR)\n$(DOLLAR)(TABLE $(DOLLAR)0)$(DOLLAR)(BR)
+DDOC_RETURNS          = $(DOLLAR)(B Returns:)$(DOLLAR)(BR)$(DOLLAR)0$(DOLLAR)(BR)$(DOLLAR)(BR)
+DDOC_SEE_ALSO         = $(DOLLAR)(B See Also:)$(DOLLAR)(BR)$(DOLLAR)0$(DOLLAR)(BR)$(DOLLAR)(BR)
+DDOC_STANDARDS        = $(DOLLAR)(B Standards:)$(DOLLAR)(BR)$(DOLLAR)0$(DOLLAR)(BR)$(DOLLAR)(BR)
+DDOC_THROWS           = $(DOLLAR)(B Throws:)$(DOLLAR)(BR)$(DOLLAR)0$(DOLLAR)(BR)$(DOLLAR)(BR)
+DDOC_VERSION          = $(DOLLAR)(B Version:)$(DOLLAR)(BR)$(DOLLAR)0$(DOLLAR)(BR)$(DOLLAR)(BR)
+DDOC_SECTION_H        = $(DOLLAR)(B $(DOLLAR)0)$(DOLLAR)(BR)$(DOLLAR)(BR)
+DDOC_SECTION          = $(DOLLAR)0$(DOLLAR)(BR)$(DOLLAR)(BR)
+DDOC_MEMBERS          = $(DOLLAR)(DL $(DOLLAR)0)
 DDOC_MODULE_MEMBERS   = $(DOLLAR)(DDOC_MEMBERS $(DOLLAR)0)
 DDOC_CLASS_MEMBERS    = $(DOLLAR)(DDOC_MEMBERS $(DOLLAR)0)
 DDOC_STRUCT_MEMBERS   = $(DOLLAR)(DDOC_MEMBERS $(DOLLAR)0)
 DDOC_ENUM_MEMBERS     = $(DOLLAR)(DDOC_MEMBERS $(DOLLAR)0)
 DDOC_TEMPLATE_MEMBERS = $(DOLLAR)(DDOC_MEMBERS $(DOLLAR)0)
 DDOC_ENUM_BASETYPE    = $(DOLLAR)0
-DDOC_PARAMS    = $(DOLLAR)(B Params:)$(DOLLAR)(BR)\n$(DOLLAR)(TABLE $(DOLLAR)0)$(DOLLAR)(BR)
-DDOC_PARAM_ROW = $(DOLLAR)(TR $(DOLLAR)0)
-DDOC_PARAM_ID  = $(DOLLAR)(TD $(DOLLAR)0)
-DDOC_PARAM_DESC  = $(DOLLAR)(TD $(DOLLAR)0)
-DDOC_BLANKLINE = $(DOLLAR)(BR)$(DOLLAR)(BR)
-
-DDOC_ANCHOR  = $(LT)a name="$(DOLLAR)1"$(GT)$(LT)/a$(GT)
-DDOC_PSYMBOL = $(DOLLAR)(U $(DOLLAR)0)
-DDOC_PSUPER_SYMBOL = $(DOLLAR)(U $(DOLLAR)0)
-DDOC_KEYWORD = $(DOLLAR)(B $(DOLLAR)0)
-DDOC_PARAM   = $(DOLLAR)(I $(DOLLAR)0)
-
-ESCAPES = /$(LT)/&amp;lt;/
-          /$(GT)/&amp;gt;/
-          /&amp/&amp;amp;/
-)
+DDOC_PARAM_ROW        = $(DOLLAR)(TR $(DOLLAR)0)
+DDOC_PARAM_ID         = $(DOLLAR)(TD $(DOLLAR)0)
+DDOC_PARAM_DESC       = $(DOLLAR)(TD $(DOLLAR)0)
+DDOC_ANCHOR           = $(LT)a name="$(DOLLAR)1"$(GT)$(LT)/a$(GT)
+DDOC_PSYMBOL          = $(DOLLAR)(U $(DOLLAR)0)
+DDOC_PSUPER_SYMBOL    = $(DOLLAR)(U $(DOLLAR)0)
+DDOC_KEYWORD          = $(DOLLAR)(B $(DOLLAR)0)
+DDOC_PARAM            = $(DOLLAR)(I $(DOLLAR)0)
+DDOC_BACKQUOTED       = $(DOLLAR)(D_INLINECODE $(DOLLAR)0)
+DDOC_BLANKLINE        = $(DOLLAR)(BR)$(DOLLAR)(BR)
+DDOC_COMMENT          = $(LT)!-- $(DOLLAR)0 --$(GT)
```

Comment:
Would be nice to keep them in the same order as in dmd.
https://github.com/D-Programming-Language/dmd/blob/312a931d03d119fa6e6af4099b5fb1beb0d71ebc/src/doc.c#L193